### PR TITLE
Build updates  (devops issue 685)

### PR DESCRIPTION
**Pull request by yakloinsteak**
## Description

Build updates and support for new CA on RDS.

## Type of change
- [x] New feature (adds functionality)
- [x] Code clean-up / housekeeping

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [ ] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: docker/app/rds-combined-ca-bundle.pem
Comment (eanders) [inline docker/app/rds-combined-ca-bundle.pem:1]: Pretty sure this file is used in other places (see database.yml, Dockerfile, sql_server_base.rb.)  Might want to make sure we update those places as well.

Comment (yakloinsteak) [reply docker/app/rds-combined-ca-bundle.pem:1]: Thanks. I'll search for it.

File: .github/workflows/build_images.yml [hunk 1]
```diff
@@ -58,10 +59,12 @@ jobs:
...
 
       - name: Cache Docker layers
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-buildx-${{ matrix.cache_key }}
+          key: ${{ runner.os }}-buildx-${{ matrix.cache_key }}-${{ github.run_id }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-${{ matrix.cache_key }}
```
Comment (yakloinsteak) [inline .github/workflows/build_images.yml:67]: This should result in better cache performance (faster builds) on average.

File: lib/rds_sql_server/sql_server_base.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class SqlServerBase < ActiveRecord::Base
   begin
     rds = Rds.new
-    cert_path = ENV['RDS_CERT_PATH'].presence || '/etc/ssl/certs/rds-combined-ca-bundle.pem'
+    cert_path = ENV['RDS_CERT_PATH'].presence || '/etc/ssl/certs/us-east-1-bundle.pem'
```
Comment (yakloinsteak) [inline lib/rds_sql_server/sql_server_base.rb:4]: Might be worth testing this out, but I'm not sure how to do that on QA.

Comment (eanders) [reply lib/rds_sql_server/sql_server_base.rb:4]: Tested.  Works.  Thanks!

**Pull request merged**